src/CMakeLists.txt

```diff
@@ -285,7 +285,7 @@ if(${WIN32})
 	add_definitions(-D__USE_MINGW_ANSI_STDIO)
 	set(LIBs ${LIBs} -ldxguid -lwinmm)
 
-	string(REPLACE "gcc" "windres" CMAKE_RC_COMPILER_INIT ${CMAKE_C_COMPILER})
```

Comment:
I’m really not sure what that’s doing here anyway. Neither should be required, and it worked fine with my toolchain file.